src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -76,10 +76,16 @@ def load_arguments(self, _):
         c.argument('environment_variables', nargs='+', options_list=['--environment-variables', '-e'], type=_environment_variables_type, help='A list of environment variable for the container. Space-separated values in \'key=value\' format.')
         c.argument('secure_environment_variables', nargs='+', type=_secure_environment_variables_type, help='A list of secure environment variable for the container. Space-separated values in \'key=value\' format.')
         c.argument('secrets', secrets_type)
-        c.argument('network_profile', network_profile_type)
         c.argument('secrets_mount_path', validator=validate_volume_mount_path, help="The path within the container where the secrets volume should be mounted. Must not contain colon ':'.")
         c.argument('file', options_list=['--file', '-f'], help="The path to the input file.")
 
+    with self.argument_context('container create', arg_group='Network Arguments') as c:
```

Comment:
Should just say `arg_group='Network'`.  Otherwise this will say "Network Arguments Arguments" :)

Reply:
Just caught that. Also is it best practice to put preview in this something like "Preview: Netowrk Arguments"?

Reply:
No, don't put preview.